__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+const db = require('../db/connection');
+const request = require('supertest');
+const app = require('../app');
+const { string } = require('pg-format');
+
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+describe("App", () => {
+    describe("GET /api/topics", () => {
+        it("Check if endpoint returns an array of topic objects", () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                expect(response.body).toBeInstanceOf(Array);
+                expect(response.body[0]).toEqual(expect.any(Object));
+            });
+        });
+        it("Check if first object matches object in db", () => {
+            return db.query(`SELECT * FROM topics;`).then((queryResult) => {
```

Comment:
We are testing our db as part of the integration testing that we are performing here so we can't use a query to the db to create an assertion from. You should remove this query and make assertions based upon what you know about the test data.